package.json

```diff
@@ -2,8 +2,11 @@
   "name": "prosemirror-math",
   "version": "0.0.1",
   "description": "Schema and plugins for first-class math support in ProseMirror.",
-  "main": "src/index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
+    "clean": "rm -rf dist",
+    "prepublishOnly": "npm run clean; tsc; cp style/math.css dist/math.css",
```

Comment:
Added something that just compiles and then copies over the css into the dist folder.